Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM node:lts-alpine
-RUN apk add --no-cache python3 make g++ tmux openssl1.1-compat
+FROM node:lts
+RUN apt-get update && apt-get -y install tmux

```

Comment:
Why leave out the equivalent apt-based `python3` and `openssl1.1-compat` packages? Are they bundled in this image?